lib/service/formula_wrapper.rb

```diff
@@ -207,24 +209,23 @@ def load_service
 
     def status_output_success_type
       @status_output_success_type ||= if System.launchctl?
-        cmd = [System.launchctl.to_s, "list", service_name]
-        output = Utils.popen_read(*cmd).chomp
-        if $CHILD_STATUS.present? && $CHILD_STATUS.success? && output.present?
+        result = system_command(System.launchctl.to_s, args: ["list", service_name])
+        output = result.stdout.chomp
+
+        if result.success? && output.present?
           success = true
-          odebug cmd.join(" "), output
```

Comment:
Does this still output the actual output of these commands or just the command being run?

Reply:
Only the command being run.

Reply:
The output is the important part though

Reply:
Do we need this debug output particularly often that it warrants outputting more than just the command? I think this is the only instance where we use `debug` this way.

Reply:
The command is actually the least relevant part of this function. The output will help developers figure out why something doesn't show up in the output.

Reply:
> Do we need this debug output particularly often that it warrants outputting more than just the command?

Yes. I've done a lot of development on `brew services` recently and this has been extremely useful.

If you're not doing `brew services` development: I have no idea why you're running this with `HOMEBREW_DEBUG` or `--debug`?

Reply:
It's not that I'm running `brew services` specifically in debug mode, I'm running everything in debug mode in CI.

Reply:
Yeh, don't do that for your dot files 😁